libs/scalajslib/src/mill/scalajslib/api/JsEnvConfig.scala

```diff
@@ -116,4 +117,140 @@ object JsEnvConfig {
+      )
+      new Playwright(options)
+
+    case class ChromeOptions(
+        headless: Boolean = true,
+        showLogs: Boolean = false,
+        debug: Boolean = false,
+        launchOptions: List[String] = defaultChromeLaunchOptions
+    ) extends Capabilities {
+      def withHeadless(value: Boolean): ChromeOptions = copy(headless = value)
+      def withShowLogs(value: Boolean): ChromeOptions = copy(showLogs = value)
+      def withDebug(value: Boolean): ChromeOptions = copy(debug = value)
+      def withLaunchOptions(value: List[String]): ChromeOptions = copy(launchOptions = value)
+    }
+    object ChromeOptions:
+      implicit def rw: RW[ChromeOptions] = macroRW
+
+    val defaultFirefoxUserPrefs: Map[String, String | Double | Boolean] =
+      Map(
+        "security.mixed_content.block_active_content" -> false,
+        "security.mixed_content.upgrade_display_content" -> false,
+        "security.file_uri.strict_origin_policy" -> false
+      )
+
+    def firefox(
+        headless: Boolean = true,
+        showLogs: Boolean = false,
+        debug: Boolean = false,
+        firefoxUserPrefs: Map[String, String | Double | Boolean] = defaultFirefoxUserPrefs
+    ): Playwright =
+      val options = FirefoxOptions(
+        headless = headless,
+        showLogs = showLogs,
+        debug = debug,
+        firefoxUserPrefs = firefoxUserPrefs
+      )
+      new Playwright(options)
+    case class FirefoxOptions(
+        headless: Boolean = true,
+        showLogs: Boolean = false,
+        debug: Boolean = false,
+        firefoxUserPrefs: Map[String, String | Double | Boolean] = defaultFirefoxUserPrefs
+    ) extends Capabilities {
+      def withHeadless(value: Boolean): FirefoxOptions = copy(headless = value)
+      def withShowLogs(value: Boolean): FirefoxOptions = copy(showLogs = value)
+      def withDebug(value: Boolean): FirefoxOptions = copy(debug = value)
+      def withFirefoxUserPrefs(value: Map[String, String | Double | Boolean]): FirefoxOptions =
+        copy(firefoxUserPrefs = value)
+    }
+    object FirefoxOptions:
+      given upickle.default.ReadWriter[String | Double | Boolean] =
+        upickle.default.readwriter[ujson.Value].bimap[String | Double | Boolean](
+          {
+            case v: Boolean => upickle.default.writeJs(v)
+            case v: Double => upickle.default.writeJs(v)
+            case v: String => upickle.default.writeJs(v)
+          },
+          json =>
+            json.boolOpt
+              .orElse(
+                json.numOpt
+              ).orElse(
+                json.strOpt.map(_.toString)
+              ).getOrElse(throw new Exception("Invalid value"))
+        )
+      given rw: RW[FirefoxOptions] = macroRW
+
+    val defaultWebkitLaunchOptions = List(
+      "--disable-extensions",
+      "--disable-web-security",
+      "--allow-running-insecure-content",
+      "--disable-site-isolation-trials",
+      "--allow-file-access-from-files"
+    )
```

Comment:
Where do these options come from? Can you link some documentation to ease future maintenence?